packages/mdc-chips/_mixins.scss

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/ripple/mixins";
+@import "@material/theme/functions";
+@import "@material/theme/mixins";
+
+@mixin mdc-chip-corner-radius($radius) {
+  border-radius: $radius;
+}
+
+@mixin mdc-chip-fill-color-accessible($color) {
+  $fill-tone: mdc-theme-tone($color);
+
+  @include mdc-chip-fill-color($color);
+
+  @if ($fill-tone == "dark") {
+    @include mdc-chip-ink-color(text-primary-on-dark);
+  } @else {
+    @include mdc-chip-ink-color(text-primary-on-light);
+  }
+}
+
+@mixin mdc-chip-fill-color($color) {
+  @include mdc-theme-prop(background-color, $color);
+}
+
+@mixin mdc-chip-ink-color($color) {
+  @include mdc-states($color);
+  @include mdc-theme-prop(color, $color);
+}
+
+@mixin mdc-chip-stroke($width: 0, $style: solid, $color: black) {
```

Comment:
Default `$width` to 1? Presumably if they're calling this, they want the stroke to be visible.